config/default.yaml

```
@@ -103,3 +103,8 @@ features:
     enabled: false
   submitEndpointForm:
     enabled: false
+  checkIssueDetailsPage:
+    # https://github.com/digital-land/submit/issues/786
+    # enabling this feature will cause the results page to present 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)